python/ray/util/accelerators/__init__.py

```diff
@@ -27,6 +27,8 @@
     "NVIDIA_TESLA_K80",
     "NVIDIA_TESLA_A10G",
     "NVIDIA_A100",
```

Comment:
so we will tag a machine with either [NVIDIA_A100, NVIDIA_A100_40G] or [NVIDIA_A100, NVIDIA_A100_80G]?

Reply:
We can tag both, so that either could be requested.